client/src/helpers/errors.ts

```diff
@@ -1,2 +1,69 @@
-// TODO: add all error checking helper functions here
-export {}
\ No newline at end of file
+//Retruns true if passed a non empty string
+function checkNonEmptyString(str: string){
+    return (str.trim() !== '');
+}
+
+//Returns true if every entry of an array is a non empty string
+function checkArrayOfStrings(strs: Array<string>){
+    let retbool = true;
+    for(let str in strs) {
```

Comment:
The "in" keyword enumerates for objects, while "of" is for arrays.
https://stackoverflow.com/questions/29285897/what-is-the-difference-between-for-in-and-for-of-statements

Reply:
Okay I fixed this